woocommerce/woocommerce
# Add Stock Notice in Single Product template

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Partially addresses https://github.com/woocommerce/woocommerce/issues/53011.

This PR adds the code to fix the `Stock Indicator` behavior under the `Simple Product` template.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Please follow the testing instructions described in PR https://github.com/woocommerce/woocommerce/pull/53319 testing instructions.
2. Then go to Appearance > Editor > Templates
3. Search for `Single Product` template.
4. Add an `Add to Cart with Options (Experimental)` block.
5. By default, the selected product type is `Simple product`, and it shows the `In stock` option. 
6. Verify that no regressions have been introduced.



### Changelog entry












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
